src/Physics/Explodinator.cpp

```diff
@@ -241,7 +241,6 @@ namespace Explodinator
 	Currently missing conduits from 1.13 */
 	static bool BlockAlwaysDrops(const BLOCKTYPE a_Block)
 	{
-		// If it's a Shulker box
```

Comment:
The floor is made out of floor

Reply:
the floor is lava and you can't changemymind

Reply:
Oh how devs will struggle without my insightful commentary